### DISCUSSION TITLE
problem facing while pushing

### DISCUSSION THREAD

**Discussion by Devesh1509** [community/community#65082] [Actions]
### Select Topic Area

Bug

### Body

fatal: unable to access 'https://github.com/......./........git/': The requested URL returned error: 403
how can I resolve this issue?

**Comment by arafajs** [comment]
The error: 403" means that you are not authorized to access the repository. There are a few possible reasons for this:

1. You may have typed the wrong username or password.
2. You may not have the correct permissions to push to the repository.
3. The repository may be private and you do not have access to it.

**Reply by Devesh1509** [reply]
the username is correct, and I m using PAT as password

**Reply by Devesh1509** [reply]
its my own repo at my personal github, do I still need permission??

**Reply by arafajs** [reply]
If the repository is private, then yes, you will still need permission to push to it. Even if it is your own repository, you will need to give yourself permission to push to it.

To do this, go to the repository settings and click on the "Collaborators" tab. Then, add your username and give yourself the "Push" permission.

Once you have done this, you should be able to push to the repository without any problems.

Here are the steps on how to give yourself permission to push to your own private repository:

1. Go to the repository on GitHub.
2. Click on the "Settings" tab.
3. Click on the "Collaborators" tab.
4. Click on the "Add collaborator" button.
5. Enter your username in the "Username" field.
6. Select the "Push" permission.
7. Click on the "Add collaborator" button.

**Reply by Devesh1509** [reply]
by following above steps I got the following massage.
 
"Repository owner cannot be a collaborator"
